tests/bluetooth/host/buf/bt_buf_get_cmd_complete/testcase.yaml

```diff
@@ -0,0 +1,20 @@
+common:
+    tags: test_framework bluetooth host
+tests:
+  bluetooth.host.bt_buf_get_cmd_complete.default:
+     type: unit
+  bluetooth.host.bt_buf_get_cmd_complete.hci_acl_flow_control:
+     type: unit
+     # enable CONFIG_BT_HCI_ACL_FLOW_CONTROL
+     extra_configs:
+      - CONFIG_BT_CENTRAL=y
```

Comment:
same comment regarding explicit setting of `CONFIG_BT_HCI_ACL_FLOW_CONTROL=y` to emphasize that is what we are testing.